README.md

```diff
@@ -53,6 +53,7 @@ Optional fields: `links` and `milestones`
 ```json
 {
   "name": "Eddie Jaoude",
+  "type": "personal",
```

Comment:
For another PR/issue: maybe we should mention about adding communities and personal accounts properly. 🙂

Reply:
Ah 🤦 this was to be in #749 I'll move the change there...

Reply:
> Ah 🤦 this was to be in #749 I'll move the change there...


Alright! But this comment was to make another PR/issue to clarify how this works.

Reply:
That will be done... I'll have a convo with Eddie, about where to add it

Reply:
and I'll make sure it is added, infact I'll add myself